minimint-bridge/src/api.rs

```diff
@@ -134,9 +135,14 @@ pub fn invoice(amount: u64, description: String) -> Result<String> {
     })
 }
 
+// TODO: impl From<Payment>
+// Do the "expired" conversion in there, too
 #[derive(Clone, Debug)]
 pub struct MyPayment {
```

Comment:
Let's call this BridgePayment to make it clear it's a type built specifically for dart's purposes